### ISSUE TITLE
Support 32-bit?

### ISSUE THREAD

**Issue by bolinfest** [facebook-atom/nuclide-prebuilt-libs#7]
I have a server that is running Ubuntu 10.04 on a 32-bit architecture that I would like to connect to with Nuclide. (I know, it's super old, which is my problem, so you can feel free to close this as wontfix.)

To get this working, first, I had to go to an external package manager to install Python 2.7 (https://askubuntu.com/questions/17841/will-python2-7-be-available-for-10-04-in-the-future), so it would be nice if the Nuclide server were pure Node to eliminate this dependency.

I was able to connect, but when I tried to open a remote terminal, I got:

```
Error starting process:
Error: Remote Error: Cannot find module '/home/mbolin/.npm-packages/lib/node_modules/nuclide/node_modules/nuclide-prebuilt-libs/pty/build/pty-v0.1.2-node-v51-linux-ia32/pty.node' processing message
```

The `pty-v0.1.2-node-v51-linux-ia32` alerted me to the fact that this is a 32-bit vm, which I believe is why the corresponding binary was not found.

**Comment by bolinfest** [comment]
More generally, it seems like `nuclide-prebuilt-libs` is primarily for Atom, no? When installing the server, it seems like it might be better (or at least an option) for the Node modules to get built from source using `node-gyp` in the traditional way.

Also, I may not be the last person on earth using a 32-bit platform because it looks like a number of Raspberry Pis do, as well: https://en.wikipedia.org/wiki/Raspberry_Pi.

**Comment by hansonw** [comment]
Yeah this repository should eventually be set up so that it properly falls back to a manual build. not a big priority though - certainly open to contributions!
